src/components/shift-schedule/ScheduleCellDialog.tsx

```diff
@@ -0,0 +1,47 @@
+import { RegisterShift } from '@/components/shift-schedule/RegisterShift';
+import { DialogHeader, DialogTitle } from '@/components/ui/Dialog';
+import { useTranslations } from 'next-intl';
+
+type ScheduleCellDialogProps = {
+  tDialog: {
+    day: string;
+    time: string;
+  };
+  members: {
+    name: string;
+  }[];
+};
+
+function ScheduleCellDialog({ tDialog, members }: ScheduleCellDialogProps) {
```

Comment:
dont pass tDialog, get the translations directly in the component isntead. The only time we need prop drilling is for client components and when the translation is not stored in the  ui or error objects.

Reply:
and it should be called just `t` for consitstency